application/extensions/admin/survey/LanguagesWidget/assets/css/LanguagesWidget.css

```
@@ -1,3 +1,8 @@
 .select2-selection__baselanguage {
     font-weight: bold;
 }
+
+.select2-container--bootstrap-5 .select2-selection--multiple .select2-selection__choice__remove:focus-visible {
+    outline: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)